### PR TITLE
Fix sqs json api folder service token

### PR DESCRIPTION
**Pull request by niksaveliev**
### Changelog category

* Not for changelog (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/core/http_proxy/http_req.cpp [hunk 1]
```diff
@@ -462,6 +462,7 @@ namespace NKikimr::NHttpProxy {
 
         public:
             void Bootstrap(const TActorContext& ctx) {
+                PoolId = ctx.SelfID.PoolID();
```
Comment (alexnick88) [inline ydb/core/http_proxy/http_req.cpp:465]: Not used variable

File: ydb/core/ymq/actor/auth_factory.h [hunk 2]
```diff
@@ -13,28 +13,29 @@ namespace NKikimr::NSQS {
 struct TAuthActorData {
     // Used by both private and public API
 
+    enum ESqsRequestFormat {
+        Xml = 0,
```
Comment (alexnick88) [inline ydb/core/ymq/actor/auth_factory.h:17]: XML, JSON - big letters

File: ydb/core/ymq/actor/auth_factory.h [hunk 3]
```diff
@@ -13,28 +13,29 @@ namespace NKikimr::NSQS {
...
     THolder<IReplyCallback> HTTPCallback;
     std::function<void(TString)> UserSidCallback;
-
     bool EnableQueueLeader;
-
     // Used by private API only
-
     EAction Action;
-
     ui32 ExecutorPoolID;
```
Comment (alexnick88) [inline ydb/core/ymq/actor/auth_factory.h:27]: Check what is set in this variable.

**Review by alexnick88 — APPROVED** [review]

File: ydb/core/http_proxy/http_req.cpp (hunk 1 above)
Comment (niksaveliev) [reply ydb/core/http_proxy/http_req.cpp:465]: It used here https://github.com/ydb-platform/ydb/pull/11244/files#diff-f22aa19d1f64f0a3be66206c685ea066c13a301980a1d3c3dde7c568821ad6abR551

**Pull request merged**